src/views/v2/widget-list.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Temporary Display of V2 List Widget
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 3ed391bf3816316609f25e7381b7f91c</sub>